plugins/scaffolder-backend/src/service/router.ts

```diff
@@ -177,17 +200,32 @@ export async function createRouter(
         }
 
         const baseUrl = getEntityBaseUrl(template);
-
-        taskSpec = {
-          baseUrl,
-          values,
-          steps: template.spec.steps.map((step, index) => ({
-            ...step,
-            id: step.id ?? `step-${index + 1}`,
-            name: step.name ?? step.action,
-          })),
-          output: template.spec.output ?? {},
-        };
+        // TODO: need to make sure that the TaskSpec is the right format here.
+        // If it's beta2 use values, beta3 uses parameters to clear that up.
```

Comment:
Isn't the TODO implemented below o_O

Reply:
Yeah maybe it shouldn't be prefixed with `TODO` 🤣